### ISSUE TITLE
Don't sort `ProjectChecker` messages by line number

### ISSUE THREAD

**Issue by Brownies** [MarketSquare/robotframework-robocop#1121]
`ProjectChecker` messages are sorted [here](https://github.com/MarketSquare/robotframework-robocop/blob/master/robocop/run.py#L156) and because of the [comparison function](https://github.com/MarketSquare/robotframework-robocop/blob/master/robocop/rules.py#L497-L502) for messages they get sorted by line number, column and then rule id.

I don't think this makes sense specifically for rules that operate on the whole project. I would prefer the checker output didn't get sorted at all in `run_project_checks()` and so the order would be up to the checker. But sorting by source path first would also be an improvement.

**Comment by bhirsz** [comment]
Good point. At this point I would even consider current behaviour as a bug.
